components/searchBar/SearchBar.module.scss

```diff
@@ -0,0 +1,20 @@
+.textBoxContainer {
+  display: flex;
+  border: solid 1px #bfc0c3;
+  max-width: 313px;
+  height: 40px;
+  border-radius: 8px;
+  background-color: white;
+  margin: 100px;
+  svg {
+    margin: 10px 16px;
+  }
+}
+
+.textBox {
+  border: none;
+  width: 100%;
+  border-radius: 8px;
+  padding-right: 10px;
+  direction: ltr;
```

Comment:
Isn't it the default?

Reply:
in styles/globals.scss the direction is rtl